arcade/application.py

```diff
@@ -934,7 +953,12 @@ class View:
     """
     Support different views/screens in a window.
     """
-
+    __slots__ = (
+        "window",
+        "key",
+        "_section_manager",
+        "section_manager",
```

Comment:
@einarf , it seems that some variables are not created in the class, so it is having issues? What should I do? I can't find anythin on it.

Reply:
Like section_manager here `conflicts with class variable`

Reply:
Slots is only for the data members. Not for properties or methods. The actual data memers is what we create instance data from.

I would skip doing slots for the Window itself. Just focus on the View. The main reason for useing slots of to save some memory per instance (no internal member dict).